subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitCategoriesIntegrationSpec.groovy

```diff
@@ -121,7 +121,22 @@ public class MyTest {
     def "re-executes test when #type is changed"() {
         given:
         resources.maybeCopy("JUnitCategoriesIntegrationSpec/reExecutesWhenPropertyIsChanged")
```

Comment:
Why are we "maybe"Copying files throughout this test, don't these files always exist?